src/renderer/components/composer/Composer.tsx

```diff
@@ -195,6 +195,9 @@ const Composer = forwardRef<
     if (!e.clipboardData.files.length) {
       return
     }
+    // when there is a file then don't paste text
+    // https://github.com/deltachat/deltachat-desktop/issues/3261
+    e.preventDefault()
```

Comment:
this prevents the `handlePaste` of the input textarea to be called, right?

isn't that missing as well in the case `if (file.path) { ddFileToDraft(file.path); return }` below?

if so, maybe better call ` preventDefault()` at the end of the event handler once, so all now and future cases are caught

Reply:
> there is no benefit in calling it multiple times

but the the `preventDefault` in the try() block is probably also superfluous

Reply:
yeah it is, looks like I forgot to remove it